test/test_data/apache_http_server.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "httpd",
```

Comment:
This test is failing:
```
>           assert product in list_products
E           AssertionError: assert 'httpd' in {'http_server'}
```

That's basically saying "we found http_server but you told us to find httpd" 

It looks from your checker above that the product name in the NVD database is http_server, so that's probably what should be here in the "product" field?